net/prometheus/Portfile

```diff
@@ -43,30 +41,22 @@ set prom_share_dir  ${prefix}/share/${name}
 set prom_log_dir    ${prefix}/var/log/${name}
 set prom_log_file   ${prom_log_dir}/${name}.log
 
-set promu_distname  promu-${promu_version}.darwin-amd64
-set promu_distfile  ${promu_distname}${extract.suffix}
-
-distfiles           prometheus-${version}${extract.suffix}:main \
-                    ${promu_distfile}:promu
+distfiles           prometheus-${version}${extract.suffix}:main
 
 checksums \
   prometheus-${version}${extract.suffix} \
```

Comment:
Can we remove `prometheus-${version}${extract.suffix}` in `checksums` now too?

Reply:
Yes, done.